src/form/state/FormFieldDeclaration.test.ts

```diff
@@ -0,0 +1,78 @@
+import { FormFieldDeclaration } from './FormFieldDeclaration.model';
+
+const createDeclaration = (arg: Partial<FormFieldDeclaration>): FormFieldDeclaration => ({
+  ...arg,
+  id: '95c4f597-99ab-4938-9c78-c4d81b75033b',
+  label: 'example',
+  name: 'example',
+});
+
+describe('Form control of a form field', () => {
+  describe('form control types', () => {
+    it('can be a textbox', () => {
+      const field = createDeclaration({
+        fieldType: 'input',
+      });
+
+      // TODO: We probably need a better name that `input`
+      expect(field.fieldType).toBe('input');
```

Comment:
Hmmm maybe `textinput` and `textarea`? Should we also have input types for `email` etc.?

Reply:
Or should the `input` be for text, numeric and email inputs together?